ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -445,8 +448,9 @@ export const policyConfigurationDescriptor: Descriptor[] = [
     {
         label: 'CVE',
         name: 'CVE',
-        longName: 'Common Vulnerabilities and Exposures (CVE) identifier',
-        negatedName: `Common Vulnerabilities and Exposures (CVE) identifier doesn't match`,
+        shortName: 'CVE',
+        longName: 'CVE identifier is',
+        negatedName: `CVE identifier doesn't match`,
```

Comment:
For future follow up: ask tech doc team whether there is IBM or Red Hat guideline about contractions for global technical English.